src/backend/aspen/workflows/nextstrain_run/build_plugins/type_plugins.py

```diff
@@ -279,20 +279,36 @@ def apply_filters(config, subsampling, template_args):
         include_arguments_in_filters = True
         lineage_field = "pango_lineage"
 
-    filter_map = {"filter_start_date": "min_date", "filter_end_date": "max_date"}
-    for filter_name, yaml_key in filter_map.items():
-        value = template_args.get(filter_name)
-        if not value:
-            continue  # This filter isn't set, skip it.
```

Comment:
I don't fully understand how this `if not value: continue` differs from the below `if pango_lineages:`